tremor-script/Cargo.toml

```diff
@@ -64,6 +65,7 @@ xz2 = "0.1"
 
 [build-dependencies]
 lalrpop = "0.19"
+chrono-tz = "0.8"
```

Comment:
do we need this twice as it's already in the deps?

Reply:
yeah, we need it at build time. Normal dependencies aren't available at build time (in build.rs).

Reply:
oh darn, thanks I didn't knew that :)